corehq/apps/users/static/users/js/mobile_workers.js

```diff
@@ -112,6 +113,30 @@ hqDefine("users/js/mobile_workers",[
             });
         });
 
+        self.sendConfirmationEmail = function () {
+            var urlName = 'send_confirmation_email';
+            var $modal = $('#confirm_' + self.user_id());
+
+            $modal.find(".btn").addSpinnerToButton();
+            $.ajax({
+                method: 'POST',
+                url: initialPageData.reverse(urlName, self.user_id()),
+                success: function (data) {
+                    $modal.modal('hide');
+                    if (data.success) {
```

Comment:
What would you think of also setting `is_account_confirmed` to true (even though it isn't really), or adding a flag like `email_sent`, to make it possible to see which users have gotten emails? Even if it only lasts until page reload, it might be helpful if someone needs to send emails to a bunch of users.

Reply:
Yeah this is a good idea - I think I'd favor the latter for clarity. Will put on the backlog for a follow up